Instructions for building in Windows.

gradiuscypher opened the pull request:
I tested this instructions while building the client on my laptop. I was able to successfully build a working EXE.

The one thing that might be missing is installing WinPcap (the software, not the developer files), as I've already had that installed due to Wireshark. Please let me know if I should add this in as well.

All feedback is appreciated.

Regner commented on building_in_windows.md:
```diff
@@ -0,0 +1,73 @@
+Grab the Microsoft Windows installer from the Download Link page. Install via the MSI. After installation, open up a command prompt and run `go version` to ensure a proper response.
+
+### Testing Go
+[Go Getting Started](https://golang.org/doc/install)
```
Probably best to link directly to the Windows section: https://golang.org/doc/install#windows.

Regner commented on building_in_windows.md:
````diff
@@ -0,0 +1,73 @@
+Using Git Bash
+
+```
+go get github.com/google/gopacket
````
We shouldn't be telling people to use `go get`. Instead people should be using Glide. See the dev instructions in the readme.

Regner commented on building_in_windows.md:
```diff
@@ -0,0 +1,73 @@
+go get github.com/google/gopacket
+go get github.com/google/gopacket/pcap
+go get github.com/regner/albionmarket-client/assemblers
+go get github.com/regner/albionmarket-client/utils
```
Also the reason you had to do `go get github.com/regner/albionmarket-client/assemblers` is because you cloned the repo to the wrong place.

You should be cloning the repo to `$GOPATH/src/github.com/regner/albionmarket-client`. That way if you make changes to the utils or assemblers it will actually use that source code.

gradiuscypher commented:
@Regner : I updated the README to include a link to the Windows guide, and I've cleaned up the steps around using Glide instead of go get. Thanks for the clarification earlier!

NullSoldier commented:
LGTM
